Install.md

```diff
@@ -66,7 +66,12 @@ If you can't use any of the package managers, you can also download [git-credent
     ```
     git-credential-manager install
     ```
-
+7. If the last command failed because Java couldn't be found make sure you have the `JAVA_HOME` variable set correctly in your environment.
+    1. Run `which java` to find out where Java is installed, this will show e.g. `/usr/bin/java`.
+    2. Run `echo $JAVA_HOME` to find out if JAVA_HOME is set, in our example case this should be `/ùsr`.

```

Comment:
`JAVA_HOME` should reference the directory where the `JRE` is installed. For example `JAVA_HOME=/usr/lib/jvm/jdk1.8.0_20`.